opea-comps/mega-service-new/chat.py

```
@@ -0,0 +1,116 @@
+from comps import ServiceOrchestrator,MicroService,ServiceRoleType,ServiceType
```

Ruff (0.8.2)

`comps.ServiceType` imported but unused

Remove unused import: `comps.ServiceType`

(F401)